src/components/Login/login.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { useState,useContext } from 'react';
+import { styles } from './styles'
+import { View,Text,TextInput,TouchableOpacity } from 'react-native';
+import { compose } from 'recompose';
+import { Formik, FormikProps } from 'formik';
+import { handleTextInput,withNextInputAutoFocusInput,withNextInputAutoFocusForm } from 'react-native-formik';
+
+import { AuthNavProps } from 'src/ultis/ParamLists/AuthParamList';
+
+import { LoginSchema } from '../../ultis/YupValidation'
```

Comment:
move utils folder under common and move all common use files to under common folder